Data/OJS.Data.Models/Contests/Contest.cs

```diff
@@ -203,6 +203,12 @@ public bool ResultsArePubliclyVisible
         [NotMapped]
         public bool IsOnline => this.Type == ContestType.OnlinePracticalExam;
 
+        [NotMapped]
+        public bool IsOnsite => this.Type == ContestType.OnsitePracticalExam;
+
+        [NotMapped]
+        public bool IsExam => this.IsOnline || this.IsOnsite;
```

Comment:
IsOnline and IsOnsite are not very explanatory since they actually refer to exams, but the names don't suddgest that. Thats ratter a comment tho, you can ignore it if it requires too much refactoring.